components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -7869,7 +7869,8 @@ public static String constructNewSearchQuery(String query) throws APIManagementE
 
         // sub context and doc content doesn't support AND search
         if (inputSearchQuery != null && inputSearchQuery.contains(" ") && !inputSearchQuery
-                .contains(APIConstants.TAG_SEARCH_TYPE_PREFIX4)) {
+                .contains(APIConstants.TAG_SEARCH_TYPE_PREFIX4) && (!inputSearchQuery
```

Comment:
shall we rename this constant to `TAG_COLON_SEARCH_TYPE_PREFIX`? 
we can rename other similar constants too similar way. 
TAGS_SEARCH_TYPE_PREFIX
TAG_EQ_SEARCH_TYPE_PREFIX etc.

Reply:
Done